frontend/public/components/pod.tsx

```diff
@@ -644,9 +646,50 @@ export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => {
     (status === 'CrashLoopBackOff' || status === 'ErrImagePull' || status === 'ImagePullBackOff') &&
     containerStatusStateWaiting
   ) {
+    let footerLinks: React.ReactNode;
+    let headerTitle = '';
+    if (status === 'CrashLoopBackOff') {
+      headerTitle = t('public~Pod crash loop back-off');
+      const containers: ContainerSpec[] = pod.spec?.containers ?? [];
+      footerLinks = (
+        <div>
+          {t(
+            'public~CrashLoopBackOff indicates that the application within the container is failing to start properly.',
+          )}
+          <br />
+          <br />
+          {t('public~To troubleshoot, view logs and/or events and then debug in terminal.')}
+          <br />
+          <br />
+          <Link to={`${resourcePath('Pod', pod.metadata.name, pod.metadata.namespace)}/logs`}>
+            {t('public~View logs')}
+          </Link>
+          &emsp;
+          <Link to={`${resourcePath('Pod', pod.metadata.name, pod.metadata.namespace)}/events`}>
+            {t('public~View events')}
+          </Link>
+          <br />
+          <Divider className="co-divider-padding" />
+          {containers.map((container) => (
```

Comment:
Do we only want to show crash looping containers here?

Reply:
Add logic to only show crashloop containers